src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -217,6 +226,28 @@ CHIP_ERROR GetManualPairingCode(std::string & aManualPairingCode, chip::Rendezvo
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR GetLongManualPairingCode(std::string & aManualPairingCode, chip::RendezvousInformationFlags aRendezvousFlags)
+{
+    chip::SetupPayload payload;
+
+    payload.commissioningFlow = chip::CommissioningFlow::kCustom;
```

Comment:
Ideally, should this be done after the call to `GetSetupPayload`?

Reply:
yes will update